src/modules/game_module.rb

```diff
@@ -0,0 +1,37 @@
+require_relative '../classes/game'
+require_relative '../classes/author'
+require_relative './author_module'
+
+module GameModule
+  def find_games
+    puts 'Game List : '
+    if @games.empty?
+      puts "\nNo  games added yet. Please add some  . . . "
+    else
+      @games.each_with_index do |game, index|
+        puts "[Game #{index + 1}]. Multiplayer : #{game.multiplayer},+
+        Publish Date : #{game.publish_date}, Last Played Date : #{game.last_played_at}"
```

Comment:
- [OPTIONAL] When listing the Games, I would advise you to remove the `+` sign at the end of the publisher's name so as not to confuse users about its relevance. 👍